akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/JoinConfigCompatCheckClusterShardingSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Ideally wanted to share test code with akka.cluster.sharding's join compatibility checker test. Not really doable with the test deps, and there was nothing shared that makes sense ATM to house it either.

Reply:
Typed cluster tests do have untyped cluster tests on the classpath or was it something more specific you needed for sharing code?